ConvertFuncs.ahk

```diff
@@ -960,11 +991,13 @@ Convert(ScriptString)
    if (SubStr(ScriptOutput, -2) = "`r`n")
       ScriptOutput := SubStr(ScriptOutput, 1, -2)
 
-   ; Add Brackets to Hotkeys
-   ScriptOutput := AddBracket(ScriptOutput)
-
-   ; Update Goto Label when Label is converted to a func
-   ScriptOutput := UpdateGoto(ScriptOutput)
+   ; 2024-06-26 andymbody - CHANGED to support masking
+   ; these will be performed in new Convert() when masking mode is ON
+   if (finalize)
```

Comment:
I plan on adding more fixes that require similar treatment the functions in this if statement, when is finalize set and when should I be using it

Reply:
See response below...

Reply:
I'm curious to see how you use it.

Keep in mind that the code may need to be pre-masked to mask comments, and strings prior to any other masking. Currently this is hidden behind the curtain of the functions in MaskCode.ahk. This may not be required depending on your implementation, but I thought I would mention it so you are aware.